### ISSUE TITLE
This is an amazingly helpful FAQ - thank you!

### ISSUE THREAD

**Issue by eob** [ahmetb/cloud-run-faq#112]
Hi @ahmetb. 

I don't have an issue to file. Just wanted to tell you that this page gave me more information about Cloud Run than Google's own site, and more efficiently. Thank you very much for putting it together!

**Comment by atrauzzi** [comment]
Agreed, this has helped me a fair bit in understanding Cloud Run better.  Wish the official docs were like this!
